src/hits-per-page/hits-per-page.ts

```diff
@@ -34,19 +40,19 @@ export type HitsPerPageState = {
   `,
 })
 export class NgAisHitsPerPage extends BaseWidget {
+  @Input() public items: HitsPerPageItem[];
```

Comment:
The `items` exposed and the `items` provided to the template are two different types.

Reply:
Good catch, thanks!

Reply:
fixed here: 2d7239e